src/FileSystemSources.props

```diff
@@ -0,0 +1,28 @@
+<Project>
```

Comment:
I like this! But maybe put it under shared? And `.props` doesn't seem like the right extension. Maybe `FileSystemSources.proj`? But I really do like the model over what we have today.